S/StaticArrays/Compat.toml

```diff
@@ -1,10 +1,19 @@
 ["0-0.10"]
-julia = ["0.7", "1"]
+julia = ["0.7", "1-1.10"]
```

Comment:
This doesn't match the title of the PR, you're allowing Julia v1.10, or am I missing something?

Reply:
You're right. I thought we might include the `AbstractTriangular` commit, which removed the second type parameter in v1.10, but I guess we are not (it would need to get backported), so I changed it accordingly.